docs/00b-basics/06-complementary-information.adoc

```diff
@@ -11,12 +10,10 @@ wo Sie gerne Feedback und Verbesserungsvorschläge geben können.
 // end::DE[]
 
 // tag::EN[]
-[discrete]
-=== Further information and terminology
+=== Supplementary Information, Terms, Translations
 
-To the extent necessary for understanding the curriculum, we have added definitions of required and important terms at the end of this document and to the https://github.com/isaqb-org/glossary[iSAQB glossary].
+To the extent necessary for understanding the curriculum, we have added definitions of technical terms to the https://github.com/isaqb-org/glossary[iSAQB glossary] and complemented them by references to (translated) literature.
```

Comment:
like to keep the original wording

Reply:
Well, then I have to adjust that in the 10 other repositories so that the wording is the same, everywhere.